sdl_android/src/main/java/com/smartdevicelink/protocol/WiProProtocol.java

```diff
@@ -575,7 +590,21 @@ private void handleSingleFrameMessageFrame(SdlPacket packet) {
 	public void StartProtocolService(SessionType sessionType, byte sessionID, boolean isEncrypted) {
 		SdlPacket header = SdlPacketFactory.createStartSession(sessionType, 0x00, getMajorVersionByte(), sessionID, isEncrypted);
 		if(sessionType.equals(SessionType.NAV)){
-			// TODO: Add bson tags for video service
+			SdlSession videoSession = sdlconn.findSessionById(sessionID);
+			ImageResolution desiredResolution = null;
+			VideoStreamingFormat desiredFormat = null;

```

Comment:
Instead of using these params, why don't we use a single class like the one created for a different PR, [`StreamingParameters`](https://github.com/smartdevicelink/sdl_android/pull/471/files#diff-df29f04f294a1bcce443240922d7bb69R61)